test/Microsoft.ML.Tests/OnnxSequenceTypeTest.cs

```diff
@@ -13,37 +13,55 @@
 using System.Linq;
 using System.IO;
 using Microsoft.ML.TestFramework.Attributes;
+using System;
```

Comment:
Please place "using System" at the top and sort alphabetical order as done in source files. System libraries take preference over non-system libraries like Microsoft.ML.*

Reply:
yes, will do

---
In reply to: [330344198](https://github.com/dotnet/machinelearning/pull/4272#discussion_r330344198) [](ancestors = 330344198)